my-app/src/components/CommentItem.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import styled from "styled-components";
+import {UserType} from "../pages/homePage/BlogHomeContext";
+
+type CommentType ={
+    _id?:string
+    comment:string
+    user:UserType
+}
+type CmtType = {
+    cmt:CommentType

```

Comment:
what is the reason to wrap commentType in CmtType ?

Reply:
fixed